### DISCUSSION TITLE
Missed name of variable in Constructor

### DISCUSSION THREAD

**Discussion by ShervanN** [ProfessionalCSharp/ProfessionalCSharp2021#164] [General]
Hello, 
I am back! To read the rest of the book!

Page 422, CHAPTER 16 Diagnostics and Metrics, In the constructor of NetworkService you missed writing the name of the variable httpClient

```
  public NetworkService(HttpClient, ILogger<NetworkService> logger)
    {
        _httpClient = httpClient;
        _logger = logger;
        _logger.LogTrace("ILogger injected into {service}", nameof(NetworkService));
    }
```

**Comment by christiannagel** [comment]
thanks for reporting, updated [updates](https://github.com/ProfessionalCSharp/ProfessionalCSharp2021/blob/main/Updates.md)
